helm/ignition-operator/templates/_resource.tpl

```diff
@@ -0,0 +1,24 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Create a name stem for resource names
+
+When pods for deployments are created they have an additional 16 character
+suffix appended, e.g. "-957c9d6ff-pkzgw". Given that Kubernetes allows 63
+characters for resource names, the stem is truncated to 47 characters to leave
+room for such suffix.
+*/}}
+{{- define "resource.default.name" -}}
+{{- .Release.Name | replace "." "-" | trunc 47 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "resource.psp.name" -}}
+{{- include "resource.default.name" . -}}-psp
+{{- end -}}
+
+{{- define "resource.pullSecret.name" -}}
+{{- include "resource.default.name" . -}}-pull-secret
+{{- end -}}
+
+{{- define "resource.default.namespace" -}}
+giantswarm
+{{- end -}}
```

Comment:
I just noticed this is out of alphabetical order. I'd move that after `resource.default.name`.

Reply:
Right, but I'd rather leave it like this, otherwise its another 4 PRs to re-order this.

Reply:
We have it like that in kvm-operator already :D 

For me it's the most important to have that straight in fmt.

Reply:
ok, that's done. I'll sort out the fmt as well.

Reply:
❤️